issuing_using_did.md

```diff
@@ -0,0 +1,202 @@
+###### Authors:
+
+Raymond Yeh
+
+# Identify Issuer & Issuing Document using DID
+
+## Goal
+
+As a new issuer, I would like to issue verifiable documents without using ethers to reduce cost, but yet being able to be identified via DNS or DID.
+As an existing issuer of verifiable document and transferable records, I would like to have my identity bound to a DID instead of DNS only.
```

Comment:
How would an identity bound to DID would look like during the verification ? 
- For dns we query the domaina and make sure the contract address is added
- For did ?